### DISCUSSION TITLE
New button feature, multiple cars?

### DISCUSSION THREAD

**Discussion by Maxxell** [G-Two/homeassistant-subaru#45] [Q&A] [answered]
How does the new button feature know which car to target if you don’t put the VIN in anymore?
Can u clarify how the buttons work?

**Comment by stboch** [comment, marked as answer]
@Maxxell The button is an new entity, with different buttons for each function for each car you have. 
Example:
button.(carName or VIN)_remote_start
you can simplly trigger this button press for each device as you like. Nice thing is now you can have them just added right to love card cards rather then having to create a script to trigger first. 

The services are still there for you to use the only breaking change was the need to add a climate preset for remote start function. This was do to a change made by Subaru to the API.

**Reply by Maxxell** [reply]
Thank you for the reply. 

My problem was that I hadn't updated the Subaru integration to 0.6 so the button wasn't working right. Once I realized that and updated, it is clear when setting up the button that you select the proper entity for the vehicle without inputting the VIN.

**Reply by Maxxell** [reply]
Can you clarify whether the climate preset is still needed with the button press system in v0.6 for a remote start command?

The example yaml doesn't have anything in "service data" for the remote start command but I can't get it to work.

**Reply by Maxxell** [reply]
Sorry. Just wanted to say I got it. The climate preset is now outside the coding for remote start entirely.
